### PR TITLE
Add description to send logs in mobile application

### PR DESCRIPTION
**Pull request by clochix**
This is a followup of #233
We probably need to add english screenshots.

### CODE REVIEW

**Comment by clochix** [comment]
Could someone review, merge and publish this one, please? Explaining how to send us the application logs is something I often do, so having it in the documentation would be great.
Thanks!

**Comment by clochix** [comment]
I just fixed the conflicts, you can now merge this PR ☺

File: src/documents/en/mobile/trouble.html.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+# Troubleshooting
+
+## Send debug informations
+
+When you hit a problem with the application, you can send us a message with the application logs, so we can try to understand and fix the problem.
+Here’s how to send the logs:
+
+
+ * click on the inco on top left corner and select Configuration inside the menu
```
Comment (poupotte) [inline src/documents/en/mobile/trouble.html.md]: inco -> icon ;)

**Comment by poupotte** [comment]
:+1:

**Comment by poupotte** [comment]
Thanks @clochix  :)

**Pull request merged**